src/main/java/mandel/earthquake/json/Geometry.java

```diff
@@ -0,0 +1,9 @@
+package mandel.earthquake.json;
+
+public class Geometry
+{
+    public double longitude;
+    public double latitude;
+    public double depth;
+    public double[] coordinates = {longitude, latitude, depth};
```

Comment:
This doesn't work the way you think it does.

Reply:
I thought that I had to put everything in an array because coordinates is an array in the json file. If I try to use the values directly, the latitude and longitude are 0.0, 0.0 because there is no array called coordinates. What should I do differently?

Reply:
You are never using the variables latitude, longitude and depth. They will always be 0.

You can only get values from the array.

Reply:
Oh ok, so I can get rid of the variables and just do this: 
public double[] coordinates = new double[3];
And then use geometry.coordinates[0] and geometry.coordinates[1] in my frame
Does that makes sense?